### ISSUE TITLE
DNA exists!

### ISSUE THREAD

**Issue by dermerlo** [HashLips/hashlips_art_engine#756]
I've already generated a collection with 3000 artwork with no problem with hundreds of attributes.

Now I'm trying to build a new little collection of 80. When i put the npm build command results always in multiply "DNA exists!" .
I have three folders layer with respectively 10, 1 and 10 attributes. 10x1x10 will results in 100 different artwork maximum supply, why if i generate only 80 artworks they results as DNA exist? Like the attributes are not enough for generate 80 different images.

**Comment by bolshoytoster** [comment]
@dermerlo It should still generate 80 artworks.

**Comment by dermerlo** [comment]
Yeah i figured it out. The process still work, it says dna exist when it try to match all the layers and find some identical, then pass to the next one untill reach the 80 artwork
